sdk/eventhub/Azure.Messaging.EventHubs/tests/Infrastructure/TaskExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace Azure.Messaging.EventHubs.Tests
```

Comment:
For context: To avoid giving the impression that this is encouraged for general use, these extensions have been scoped only to the test project.